include/mbedtls/md.h

```diff
@@ -113,7 +113,10 @@
 #define MBEDTLS_MD_SOME_LEGACY
 #endif
 #if defined(MBEDTLS_SHA3_C)
-#define MBEDTLS_MD_CAN_SHA3
+#define MBEDTLS_MD_CAN_SHA3_224
```

Comment:
Note: we should have tooling to catch this kind of thing. In fact, we already have 90% of it but it's no good because it doesn't shout when it finds something. See #2691 and #5389, but also #6099.